scripts/globusmonitor/globus_uploader_service.py

```diff
@@ -396,7 +396,7 @@ def notifyClowderOfCompletedTask(task):
                                          headers={'Content-Type':'application/json'},
                                          data=json.dumps(md))
 
-                        if dsmd.status_code in [500, 502, 504]:
+                        if dsmd.status_code in [104, 500, 502, 504]:
```

Comment:
This was included in another PR. Is it needed here or just a stray commit?

Reply:
accident